GameServerCore/Packets/PacketDefinitions/Requests/AutoAttackOptionRequest.cs

```diff
@@ -1,6 +1,6 @@
 ﻿namespace GameServerCore.Packets.PacketDefinitions.Requests
 {
-    public class AutoAttackOptionRequest
+    public class AutoAttackOptionRequest : ICoreRequest
     {
         public int NetId { get; }
```

Comment:
Replacing ``NetID``'s where available to their proper ``IGameObject``,``IAttackableUnit`` etc. or less ideally with ``struct NetID``.
Should be addressed in another PR probably because of complexity of such changes.

Reply:
Probably we need to change this in all packets, but this change is complex one and will change almost all the systems in lib I think.